safe-global/safe-wallet-web
# fix: get Safe address on-chain for ZK creations

## What it solves

Resolves #2370

## How this PR fixes it

This updates the `@safe-global/safe-core-sdk` to `3.3.5` [as per requested](https://5afe.slack.com/archives/C03DAGWJCR5/p1692284450682389) which fixes Safe address prediction on ZK chains.

## How to test it

Create a Safe on a ZK chain and observe the correct Safe address in the creation stepper/navigation link after indexed.

## Checklist
